osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -625,6 +631,30 @@ public SkinEditorToast(LocalisableString value, string skinDisplayName)
             }
         }
 
+        public partial class ResetConfirmDialog : DangerousActionDialog
+        {
+            public ResetConfirmDialog(Action reset)
+            {
+                HeaderText = SkinEditorStrings.RevertToDefaultDescription;
+                BodyText = SkinEditorStrings.ResetDialogue;
+
+                Icon = FontAwesome.Solid.ExclamationTriangle;
+
+                Buttons = new PopupDialogButton[]
+                {
+                    new PopupDialogDangerousButton
+                    {
+                        Text = BeatmapOverlayStrings.UserContentConfirmButtonText,
+                        Action = reset
+                    },
+                    new PopupDialogCancelButton
+                    {
+                        Text = Web.CommonStrings.ButtonsCancel,
+                    },
+                };
```

Comment:
Inheriting `DangerousActionDialog` makes zero sense if we're just going to overwrite the buttons entirely and completely not use `DeleteAction`. Like if it's *intentionally not going* to use the structure of `DangerousActionDialog`, it would be better to not inherit it and revert that entire change.

Reply:
yeah i don't know exactly what i was thinking, probably lack of thinking that caused this issue, should be fixed with the latest commit